tests/acceptance/ENFormCreateCept.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
```

Comment:
Can you add a group annotation here?
It would be nice to start grouping tests so we can run specific tests

Like this

```
/**
 * @group engagingnetworks
 */
```